packages/contracts-bedrock/src/periphery/faucet/Faucet.sol

```diff
@@ -123,7 +123,8 @@ contract Faucet {
         nonces[_auth.id][_params.nonce] = true;
 
         // Execute transfer of ETH to the recipient account.
-        SafeCall.call(_params.recipient, _params.gasLimit, config.amount, _params.data);
+        bool success = SafeCall.call(_params.recipient, _params.gasLimit, config.amount, _params.data);
+        require(success, "Failed to execute SafeCall during drip to recipient");
```

Comment:
can you add a test for this to https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/test/periphery/faucet/Faucet.t.sol?